src/internal/templates/internal/lore_wiki.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'internal/base.html' %}
+{% load i18n %}
+{% load static %}
+
+
+{% block title %}
+    {% translate "The wiki of MAKE lore" %}
+{% endblock title %}
+
+{% block header_links %}
+    <link rel="stylesheet" href="{% static 'internal/css/make_lore.css' %}">
+    <link rel="stylesheet" href="{% static 'web/css/header.css' %}"/>
+    <script defer src="{% static 'web/js/header.js' %}"></script>
+{% endblock header_links %}
```

Comment:
I think you mistakenly overrode `header_links` instead of `extra_head`, which would explain why you linked `header.css`  and `header.js` here 😅

```suggestion
{% block extra_head %}
    <link rel="stylesheet" href="{% static 'internal/css/make_lore.css' %}">
{% endblock extra_head %}
```